src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
```

Comment:
This change comes down as the file is updated in `master` to avoid having to merge it repeatedly down the line.